obspy/imaging/maps.py

```diff
@@ -144,7 +151,6 @@ def plot_basemap(lons, lats, size, color, labels=None,
     if show_colorbar:
         map_ax = fig.add_axes([ax_x0, 0.13, ax_width, 0.77])
         cm_ax = fig.add_axes([ax_x0, 0.05, ax_width, 0.05])
-        plt.sca(map_ax)
```

Comment:
There was an issue with the map being plotted inside the colorbar axes when just replacing `plt.sca()` in all occurences. This has been fixed by manually assigning the axes of the map. The plots seem to end up correct.